packages/api-derive/src/session/info.ts

```diff
@@ -13,13 +13,13 @@ import { BlockNumber, Option } from '@polkadot/types';
 import { drr } from '../util/drr';
 import { bestNumber } from '../chain';
 
-type Result0_94 = [BN, [BN, Option<BlockNumber>, BN, BN, BN]];
+type Result_pre_94 = [BN, [BN, Option<BlockNumber>, BN, BN, BN]];
 type Result = [BN, [BN, BN]];
 
 const ZERO = new BN(0);
 
 // internal helper to just split the logic - take all inputs, do the calculations and combine
-function createDerived0_94 ([bestNumber, [currentIndex, _lastLengthChange, sessionLength, lastEraLengthChange, sessionsPerEra]]: Result0_94): DerivedSessionInfo {
+function createDerived_pre_94 ([bestNumber, [currentIndex, _lastLengthChange, sessionLength, lastEraLengthChange, sessionsPerEra]]: Result_pre_94): DerivedSessionInfo {
```

Comment:
Let's have the convention everywhere `someThingWhatever_{start_spec}_{end_spec}`, e.g. we already have EventRecord_0_76, here it'd be `createDerived_0_94`

Reply:
Sorry :(

Reply:
Ah, that was the `0`! 💡

Reply:
https://github.com/polkadot-js/api/pull/1104